dev/airflow-jira

```diff
@@ -1,21 +1,22 @@
 #!/usr/bin/env python
```

Comment:
can we keep because it has no file ending?!

Reply:
What do you mean :) ?

Reply:
Sry :D I mean it has no file-ending hence we need it to be recognized as python script?

Reply:
Not really. The shebang is needed if you want to run it simply as `./airflow-jira` without python in front. It has nothing to do with file ending :).

Reply:
TIL (Today I learned) - Thank you :)